### PR TITLE
cron in Docker image

### PR DESCRIPTION
**Pull request by Alkarex**
https://github.com/FreshRSS/FreshRSS/issues/1869

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]
Looks good to me.

**Comment by Alkarex** [comment]
@kordianbruck Does that do the job for you?

File: Docker/README.md [hunk 1]
```diff
@@ -89,16 +92,49 @@ sudo docker exec --user apache -it freshrss php ./cli/list-users.php
...
 
+### 3) Cron as another instance of the same FreshRSS Docker image
+For advanced users. Offers good logging and monitoring with auto-restart on failure.
+Watch out to use the same run parameters than in your main FreshRSS instance, for database, networking, and file system.
+See cron option 1 for customising the cron schedule.
+
+```sh
+sudo docker run -dit --restart unless-stopped --log-opt max-size=10m \
+	-v $(pwd)/data:/var/www/FreshRSS/data \
+	-e 'CRON_MIN=17,37' \
```
Comment (Alkarex) [inline Docker/README.md]: Ups, that does not work in that case. Fix needed.

Comment (Alkarex) [reply Docker/README.md]: Fixed in https://github.com/FreshRSS/FreshRSS/pull/1871/commits/ecf366da77060d34399fe860227232781427e4e2

**Comment by kordianbruck** [comment]
@Alkarex this looks great! thanks!

**Pull request merged**